presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -371,7 +370,9 @@ private ConnectorAccessControl createAccessControl(String fileName)
 
     private static void assertDenied(ThrowingRunnable runnable)
     {
-        assertThrows(AccessDeniedException.class, runnable);
+        assertThatThrownBy(runnable::run)
+                .isInstanceOf(AccessDeniedException.class)
+                .hasMessageStartingWith("Access Denied");
```

Comment:
We should improve these tests to match `TestFileBasedSystemAccessControl`

please add a TODO here
```
// TODO test expected message precisely, as in TestFileBasedSystemAccessControl
```